### ISSUE TITLE
Ability to switch the JAVA version for a Demo Environment

### ISSUE THREAD

**Issue by rwunsch** [Adobe-Marketing-Cloud/aem-demo-machine#37]
Hello @coatpont ,

Thanks for writing, maintaining and providing this wonderful tool.
I wanted to create a pull-request at some point, but did not find the time, so I am creating an issue for now.

I would find it wonderful to be able to configure the java version for each Environment.
I am using CQ/AEM down to CQ5.4 with aem-demo-machine, as a quick way to set-up instances.
But for lower environments I need to switch the java version beforehand.

I would find it exetremely helpful to be able to configure java version 6,7,8,(9) in the variables, which I could select when creating the environment, which in turn would be used to start-up the environment.
So basically using a Full-path to start java-applications.

Does this make sense?

Let's see if this makes sense, and maybe you can guide me to the appropriate parts to create the pull request eventually.

Thanks and cheers
Robert

**Comment by coatpont** [comment]
hi Robert, I just pushed a change that will allow you to control the JVM to be used to install and to start AEM (demo.jvm) - it defaults to "java" but you can put a fully qualified path to any JVM in there (link to the java executable). 

Here's what I did to test with CQ561. In my demos folder, I created a new "cq561" folder, then in there I created a "demobuild.properties" file with the following settings:

demo.acs=false
demo.hotfixes=false
demo.osgi=false
demo.jar=cq561
demo.jvm=/Library/Java/JavaVirtualMachines/jdk1.7.0_51.jdk/Contents/Home/bin/java
demo.license=cq5

Then I started the AEM demo machine GUI and selected the cq561 environment and clicked on "rebuild" - which is recreating a demo out of these settings (taking precedence over the main build.properties and conf/build-personal.properties). 

This allowed me to run all demos on Java8 (my default java) and this instance on Java7

**Comment by rwunsch** [comment]
Thank you Bertrand!
I am trying to use the feature using WIN10 and get back to you - the first try was unsuccessful, but this might have had other reasons. 
I pulled the newest version before testing.

Getting back later!

Thanks and cheers
Robert
